src/sentry/static/sentry/app/views/organizationDashboard.jsx

```diff
@@ -207,6 +209,125 @@ const Activity = React.createClass({
   },
 });
 
+const EPH = React.createClass({
+
+  mixins: [ApiMixin, OrganizationState],
+
+  getInitialState() {
+    let until = Math.floor(new Date().getTime() / 1000);
+    let since = until - 3600 * 24;
+
+    return {
+      rawOrgData: {},
+      formattedData: null,
+      querySince: since,
+      queryUntil: until,
+      error: false
+    };
+  },
+
+  componentDidMount() {
+    this.fetchData();
+  },
+
+  // Use this so there's a standard order for y values and bar classes
+  STAT_OPTS: ['received', 'rejected', 'blacklisted'],
+
+  fetchData() {
+    let statEndpoint = this.getEndpoint();
+
+    let query = {
+      since: this.state.querySince,
+      until: this.state.queryUntil,
+      resolution: '1h',
+    };
+
+    $.when.apply($, this.STAT_OPTS.map(stat => {
+        let deferred = $.Deferred();
+        this.api.request(statEndpoint, {
+          query: Object.assign({stat: stat}, query),
+          success: deferred.resolve.bind(deferred),
+          error: deferred.reject.bind(deferred)
+        });
+        return deferred;
+      }
+    )).done(function() {
+      let rawOrgData = {};
+      for (let i = 0; i < this.STAT_OPTS.length; i++) {
+        rawOrgData[this.STAT_OPTS[i]] = arguments[i][0];
+      }
+      this.setState({
+        rawOrgData: rawOrgData,
+        formattedData: this.formatData(rawOrgData)
+      });
+    }.bind(this)).fail(function() {
+      this.setState({error: true});
+    }.bind(this));
+  },
+
+  getEndpoint() {
+    return `/organizations/${this.props.params.orgId}/stats/`;
+  },
+
+  formatData(rawData) {
+    // Do this grossness to make sure data is zero filled
+    let allXValues = {};
+    let valueLookup = {};
+    this.STAT_OPTS.forEach(stat => {
+      valueLookup[stat] = {};
+      rawData[stat] && rawData[stat].forEach(point => {
+        allXValues[point[0]] = null;
+        valueLookup[stat][point[0]] = point[1];
+      });
+    });
+    allXValues = Object.keys(allXValues);
+
+    let chartData = [];
+    allXValues.forEach(x => {
+      let point = {x: +x, y: []};
+      // convert received --> accepted
+      let acceptedY = valueLookup.received[x] - valueLookup.rejected[x] - valueLookup.blacklisted[x];
+      point.y.push(acceptedY);
+      ['rejected', 'blacklisted'].forEach(stat => {
+        let yVal = valueLookup[stat][x] || 0;
+        point.y.push(yVal);
+      });
+      chartData.push(point);
+    });
+
+    chartData.sort((a, b) => {
+      if (a.x < b.x) {
+        return -1;
+      }
+      if (a.x > b.x) {
+        return 1;
+      }
+      return 0;
+    });
+
+    return chartData;
+  },
```

Comment:
I definitely feel like this is a lot of complicated code (a lot of branching) that really, really could use some cursory tests. 

Remember: it's for the next person who comes along and has to change this.